assets/js/script.js

```diff
@@ -52,6 +54,18 @@ function pageLoad() {
 		setTimeout(() => fadeIn(document.getElementById('cookie-container')), 1000);
 	};
 
+	const username = getCookie("user");
+	if (username) {
+		if (isMainPage) document.getElementById("username-content").innerHTML = "Hallo, <span class='accent'>" + username + "</span>!";
+		document.getElementById("username-header").innerText = username;
+		document.getElementsByClassName("accdropdown-content")[0].innerHTML = '<a href="/logout" translation="global.logout">Abmelden</a><a href="/dashboard/user">Eigene Daten ansehen</a>';
```

Comment:
Shouldn't this be translated?

Reply:
Translations coming soonTM in another update